packages/consumption/src/modules/requests/itemProcessors/shareAttribute/ShareAttributeRequestItemProcessor.ts

```diff
@@ -90,6 +90,9 @@ export class ShareAttributeRequestItemProcessor extends GenericRequestItemProces
                     );
                 }
             }
+            if (requestItem.thirdPartyAddress) {
+                return ValidationResult.error(ConsumptionCoreErrors.requests.invalidRequestItem("Identity attributes are not allowed to have a thirdPartyAddress"));
```

Comment:
Rephrase please, as it sounds like it's about a (non-existing) property of the IdentityAttribute and not the ShareAttributeRequestItem here, for example: `"When sharing a RepositoryAttribute, no thirdPartyAddress may be specified."`